.golangci.yml

```
@@ -7,4 +7,5 @@ linters:
     - whitespace
 output:
   print-issued-lines: true
-  out-format: github-actions
+  format: 
+    - github-actions
```

Based on https://github.com/golangci/golangci-lint/blob/master/.golangci.reference.yml#L4108 I think this should be
```suggestion
  formats: 
    - format: github-actions
```